zerver/tests/tests.py

```diff
@@ -2132,7 +2132,7 @@ def test_open_realm_logic(self):
         mit_realm.save()
         self.assertTrue(completely_open(mit_realm.domain))
         self.assertEquals(get_unique_open_realm(), None)
-        with self.settings(SYSTEM_ONLY_REALMS={"zulip.com"}):
+        with self.settings(SYSTEM_ONLY_REALMS={"zulip.com", "analytics.test"}):
```

Comment:
`get_unique_open_realm` is expecting there to be exactly one open realm (that's not a SYSTEM_ONLY_REALM):
```
def get_unique_open_realm():
    realms = Realm.objects.filter(deactivated=False)
    realms = realms.exclude(domain__in=settings.SYSTEM_ONLY_REALMS)
    if len(realms) != 1:
        return None
    ...
```
This test passes locally for me, but fails on travis, I'm guessing because `analytics.test` is in the database at the time this test happens to be run.